### DISCUSSION TITLE
Welcome to Sunbird UCI Community Discussions!

### DISCUSSION THREAD

**Discussion by aayushkedia95** [samagra-comms/community#3] [Announcements]
👋 Welcome to Discussions on Sunbird UCI. Please go through this post to understand how to engage via this forum.

We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about
- Share ideas
- Engage with other community members
- Welcome others and are open-minded, remember that this is a community we build together 💪

Please post questions in the right category:

- Announcement - Maintainers and admins will share the announcements related to the community in this category
- Developer Setup and Troubleshooting - This category is to help people who are trying to set up Sunbird UCI on their local
- General Discussion - This category can be used to discuss anything related to Sunbird UCI
- Knowledge base - This category is for the Sunbird UCI maintainers who share information or updates pertaining to UCI
- New features - This category is for people to discuss potential new capabilities, and design to solve specific problems
- Meta - This category is for all the contributors to bring up issues related to the process of community collaboration and contribution. You can highlight something that is not working and also propose changes to the process.
- Temp - If there are discussions that are more of here and now and do not add value to the larger audience, in the long run, for example - requesting someone for a meeting, feel free to use this category. The discussion threads of this category will be deleted periodically

If you are unable to resolve your issues over the forums and need to engage in a synchronous interaction - please record the interaction, and share the link to it in the corresponding discussion. This is with the intent of ensuring all communication happens in the open, and can be reused by those who come after you and face a similar challenge.

Finally, be nice to one another, and forgive ourselves for our slip-ups. We'll get through this together.

**Comment by Athiqur-r** [comment]
Dear SunBird Team,

I am writing to seek your help regarding the setup of the uci-api project available at https://github.com/samagra-comms/uci-apis. We are facing some issues with the code, especially with the file "schedulerService.js" in the path "sunBirdUci\backendSunBird\uci-apis\src\service\schedulerService.js". Currently, we are encountering the following error: "error FetchError: request to http://localhost:8888/ failed, reason: connect ECONNREFUSED ::1:8888".

As we plan to use this plugin in our application, we would like to set up everything through Docker and run the application. However, we are facing some difficulties in doing so, and we would appreciate your assistance in resolving the issue.

Could you please guide us in setting up the uci-api locally through Docker? We have already cloned the repository, but we are not sure how to proceed further. Any help regarding this would be highly appreciated.

Thank you for your time and support.

**Comment by raghavinnomick** [comment]
Dear SunBird Team,

I am writing to seek your help regarding the setup of the uci-api project available at https://github.com/samagra-comms/uci-apis. We are facing some issues with the code, especially with the file "schedulerService.js" in the path "sunBirdUci\backendSunBird\uci-apis\src\service\schedulerService.js". Currently, we are encountering the following error: "error FetchError: request to http://localhost:8888/ failed, reason: connect ECONNREFUSED ::1:8888".

As we plan to use this plugin in our application, we would like to set up everything through Docker and run the application. However, we are facing some difficulties in doing so, and we would appreciate your assistance in resolving the issue.

Could you please guide us in setting up the uci-api locally through Docker? We have already cloned the repository, but we are not sure how to proceed further. Any help regarding this would be highly appreciated.

Thank you for your time and support.
